docs/token-guides/wsteth-bridging-guide.md

```diff
@@ -0,0 +1,353 @@
+
+### The Lido DAO recognition recommendations by the NEW
+
+The recommendations from R-5 to R-8 are crucial for the recognition of the bridged wstETH endpoints by the Lido DAO.
+
+The recommendations starting from R-9 are not crucial but significantly contribute to the likelihood of the Lido DAO recognition.
+
+#### R-5: Bridging L1 Lido DAO decisions
+
+Control over the contracts endpoints must be passed to the Lido DAO Agent on L1. The proposed method of doing this is to use a dedicated governance executor contract on L2.
+
+Examples:
+- [`OptimismBridgeExecutor`](https://optimistic.etherscan.io/address/0xefa0db536d2c8089685630fafe88cf7805966fc3);
+- [Bridge executor on Base](https://basescan.org/address/0x0E37599436974a25dDeEdF795C848d30Af46eaCF) - reused `OptimismBridgeExecutor` contract;
+- [`ZkSyncBridgeExecutor`](https://explorer.zksync.io/address/0x13f46b59067f064c634fb17e207ed203916dccc8#contract)
+- [`LineaBridgeExecutor`](https://lineascan.build/address/0x74Be82F00CC867614803ffd7f36A2a4aF0405670)
+
+For more examples, see Governance Bridge Executors at https://docs.lido.fi/deployed-contracts/#lido-on-l2. The contracts originate from [Aave Governance Cross-Chain Bridges](https://github.com/aave/governance-crosschain-bridges) and can be found at https://github.com/lidofinance/lido-l2/ and [PRs](https://github.com/lidofinance/governance-crosschain-bridges/pulls).
+
+#### R-6: Dedicated upgradable bridge instances
+
+Deploy dedicated upgradable instances of bridges on L1 and L2.
+This is required to lay the foundation for the emergency capabilities (R-7) and for possible bridging of rebasable stETH. For more details on why, see for section [Why this guide needed](#why-is-this-guide-needed). For the architecture outline, see section [Reference architecture and permissions setup](#reference-architecture-and-permissions-setup).
+
+#### R-7: Pausable deposits and withdrawals
+
+To provide the capability to react fast and reduce losses in case of a security contingency, depositing and withdrawing should be pausable. Namely:
+
+- L1 bridge has pausable and resumable deposits;
+- L2 bridge/token has pausable and resumable withdrawals.
+
+The capability to pause should be granted both to the DAO Agent instance (or its L2 representation) and the dedicated Emergency Multisig which is [ratified by the Lido DAO](https://snapshot.org/#/lido-snapshot.eth/proposal/0xfe2a6a6506a642b616118363bc29aa83dd9ef2ec80447bb607a8f52c0a96aed0). The permission to resume is proposed to be granted only to the Lido DAO Agent (or its L2 representation).
+
+To reduce the power of the multisig beyond necessary, it is proposed to use "Gate Seals" mechanic. The mechanic limits the pause duration and restricts the capability to pause to a single time. To grant the capability repeatedly, the Lido DAO vote is required. The mechanic has been implemented, e.g., for withdrawals in Lido protocol on Ethereum in two parts:
```

Comment:
> To reduce the power of the multisig beyond necessary

This doesn't sound right. Consider rephrasing, e.g.

"To curb the multisig's power, it is proposed..."

Reply:
> to a single time

single use